package.json

```diff
@@ -72,7 +72,7 @@
       "^.+\\.svg$": "<rootDir>/svgTransform.js"
     },
     "transformIgnorePatterns": [
-      "/node_modules/(?!(web-ifc|web-ifc-three|web-ifc-viewer|three)/)"
+      "/node_modules/(?!(@pablo-mayrgundter\/cookies.js|web-ifc|web-ifc-three|web-ifc-viewer|three)/)"
```

Comment:
Why is this needed?

Reply:
Without this, Jest expects the module to be already transformed into a CommonJS artifact. Since the packages listed in this regex are ES6 modules, Jest errors out on the ES6 import/export syntax.

This directive tells Jest to transpile these matching modules prior to executing the tests.

https://jestjs.io/docs/configuration#transformignorepatterns-arraystring